static/views/welcome.html

```diff
@@ -9,6 +9,7 @@
 
 <body>
   <div id="welcome-root"></div>
+  <script src="/assets/js/html5-qrcode.min.js"></script>
```

Comment:
do we need this AND the npm package as dependency?

Reply:
Currently only the script tag. So it could be left out. I left the dependency for a couple of reasons:
- The package might support module in the future (that way we can import directly from within react).
- We could create a build task that copies html5-qrcode.min.js from the node_modules, instead of the currently manually included one.

Reply:
👍 adding a TODO there